inference-engine/src/transformations/include/transformations/control_flow/unroll_if.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include <transformations_visibility.hpp>
+#include <vector>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API UnrollIf;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief Unrolls the body of the If layer if condition is const.
+ */
+
+class ngraph::pass::UnrollIf : public ngraph::pass::FunctionPass {
+public:
+    NGRAPH_RTTI_DECLARATION;
+    bool run_on_function(std::shared_ptr<ngraph::Function> f);
```

Comment:
Maybe this function should be marked as `override`?

Reply:
fixed